clear/main.py

```diff
@@ -216,6 +216,17 @@ def clear_wm_pg_shared():
     command = 'watermarkstat -c -t pg_shared'
     run_command(command)
 
+@priority_group.group()
+def drop():
```

Comment:
Please add the check so that clear can be executed only by 'root' . check the watermark group for reference

Reply:
There is one problem:
For executing **sonic-clear priority-group drop counters** we use **runner.invoke**. But  there is no ability to run it with 'root' privileges  (I could not find appropriate solution). In tests that related to watermarks  there are no tests with  checking "clear counters" functionality.  
Possible solution:
- accept it problem (add **if os.geteuid() != 0** check) and add checking to "Root privileges are required for this operation" message  (can be temporary solution)
- do not add checking  **if os.geteuid() != 0** and leave as is
- try to find another way to run  command **sonic-clear priority-group drop counters** with root privilege 

Currently first proposal done locally

Reply:
Can you use a similar solution to bypass the root check for unit tests?
https://github.com/Azure/sonic-utilities/blob/master/scripts/ecnconfig#L94